examples/hf/pippy_camemBert.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 4 pippy_camemBert.py
+
+import argparse
+import os
+
+import torch
+import torch.distributed as dist
+
+from pippy.IR import Pipe, PipeSplitWrapper, annotate_split_points
+from pippy.PipelineStage import PipelineStage
+
+from transformers import CamembertForMaskedLM, CamembertConfig
+
+from hf_utils import generate_inputs_for_model, get_number_of_params
+
+
+def add_split_points(camembert, nranks):
+    layers_per_rank = camembert.config.num_hidden_layers // nranks
+    for i in range(1, nranks):
+        annotate_split_points(
+            camembert, {f"roberta.encoder.layer.{i * layers_per_rank}": PipeSplitWrapper.SplitPoint.BEGINNING})
+
+
+def run(args):
+    # Model configs
+    config = CamembertConfig()
+    print("Using device:", args.device)
+
+    # Create model
+    model_class = CamembertForMaskedLM
+    model_name = "CamembertForMaskedLM"
+    camembert = model_class(config)
+    camembert.to(args.device)
+    camembert.eval()
+    if args.rank == 0:
+        print(camembert.config)
+        print(f"Total number of params = {get_number_of_params(camembert) // 10 ** 6}M")
```

Comment:
minor - have get num params return in M by default via default arg, instead of doing same hand math in every example? 
can put MB, GB, etc. as constants in HF_utils rather than having 10**6, 10**9 etc. in the code?